backend/tests/integration/records/serializers/json/test_literature.py

```diff
@@ -900,3 +900,29 @@ def test_record_link_to_hepdata_in_detail_view(inspire_app):
         response.json["metadata"]["external_system_identifiers"]
         == expected_response_metadata
     )
+
+
+def test_record_returns_linked_book(inspire_app):
+    parent_record = create_record("lit")
+
+    expected_linked_book = {
+        "self": f"http://localhost:5000/literature/{parent_record['control_number']}",
```

Comment:
I think it's more consistent to heave `record: { $ref: .... }` instead, since we always wrap `$ref` links.

Reply:
:+1: